releasenotes/notes/create-diagnose-datadog-connectivity-subcommand-53403e7599a38084.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
```

Comment:
If the feature is not implemented yet, we probably don't want to announce it in the changelog. We should either remove this (and use `no-changelog` label), or implement the command before merging this PR.